src/app/admin/packingSlotsTable/PackingSlotActions.ts

```diff
@@ -0,0 +1,162 @@
+import supabase from "@/supabaseClient";
+import { DatabaseError } from "@/app/errorClasses";
+import { GridRowId } from "@mui/x-data-grid";
+import { PackingSlotRow } from "@/app/admin/packingSlotsTable/PackingSlotsTable";
+
+export interface DBPackingSlotData {
+    primary_key?: string;
+    name: string;
+    is_shown: boolean;
+    order: number;
+}
+
+export interface DataGridPackingSlotRow {
+    id: string;
+    name: string;
+    is_shown: boolean;
```

Comment:
I think we should rename this to `isShown` as this interface is for internal use on the website, not for the database.